taskcluster/ci/config.yml

```diff
@@ -129,6 +129,11 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/dynamic-probe-telemetry-extension
             default-ref: main
             type: git
+        searchdefaultaddon:
+            name: "Reset Search Default extension"
+            project-regex: searchdefaultaddon$
+            default-repository: https://github.com/mozilla-extensions/restore-search-defaults
+            default-ref: main
```

Comment:
The decision task is currently failing with `[task 2020-12-11T19:34:49.775Z] KeyError: u'type'`.
We need `type: git` here, since taskgraph supports both hg and git.